### PR TITLE
FEAT: added auth to private routes

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
**Files To check:**

- Auth
- Navigation
- AuthContext
- login
- logout
- signup

**Screenshots**
image

image

image

### CODE REVIEW

File: src/pages/LogoutPage/logout.css [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+.checkbox-signup {
+    display: flex;
+    width: 30rem;
+    gap:1rem;
+    align-items: baseline;
+    padding: 1rem;
+    font-size: 1.5rem;
+    font-weight: 500;
+} */
```
Comment (Naman-Saxena1) [inline src/pages/LogoutPage/logout.css]: Too much commented code, maybe you can cross check if you need it

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
In some files I found decent amount of commented code, so maybe you can reconsider if you really it
Also, when I open private tab like watch later without login, it is taking me to signing but is not showing any signing form, you can cross check that

PrivateRoute

**Review by prankurpandeyy — APPROVED** [review]

**Comment by prankurpandeyy** [comment]
Look at changes suggested by naman

File: src/App.js [hunk 2]
```diff
@@ -17,24 +13,64 @@ import {
...
+          }
+        />
+        <Route
+          path="/playlist"
+          element={
+            <Auth>
+              <PlaylistPage />
+            </Auth>
+          }
+        />
```
Comment (SinchanaMS) [inline src/App.js:62]: You can make use of Outlet to get rid of repeated code for the children here. 
Let me know if you want to implement it, I can help!

**Comment by Adarsh-Patel-Dev** [comment]
> In some files I found decent amount of commented code, so maybe you can reconsider if you really it Also, when I open private tab like watch later without login, it is taking me to signing but is not showing any signing form, you can cross check that
> 
> PrivateRoute

Actually there was a typo in link name. Now it is working

File: src/pages/LogoutPage/logout.css (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply src/pages/LogoutPage/logout.css]: Yeah I forgot to remove commented code🙂

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Functionality is now working fine
I suggest you increase opacity of background, I understand it is for styling but contrast ratio is too low 
Screenshot 2022-05-19 162829

**Review by SinchanaMS — APPROVED** [review]
I couldn't find any major suggestions. 
You can keep consistent spacing in the code and get rid of logs and commented code.
Everything else looks fine!

File: src/App.js (hunk 2 above)
Comment (Adarsh-Patel-Dev) [reply src/App.js:62]: Thanks for suggesting outlet, I will try to implement it.

**Pull request merged**